src/core/src/op/loop.cpp

```diff
@@ -143,10 +143,6 @@ void op::v5::Loop::validate_and_infer_types() {
             input_offset--;
         }
     }
-    // input_offset < 0 means that there are several duplications of external_port_id
```

Comment:
Looks like this check is valid. I guess it is better to change 'input_offset' from unsigned `size_t` to signed `int64_t` and keep this check. Or cast to 'int64_t' inside this check.

Reply:
Ok I will change the type of variable

Reply:
Fixed